### ISSUE TITLE
LRC alternative hosts

### ISSUE THREAD

**Issue by greplova** [virtualscienceforum/virtualscienceforum#339]
I had trouble starting LRC as non main virtualscienceforum user. There used to be a "Start meeting" button - but now I couldn't find and all links took me back to the registration page. Can we check we have alternative hosts assigned correctly in the new LRC meeting creation? Thanks!

**Comment by akhmerov** [comment]
@everthemore we should get all admin users and add them to the newly created meeting as alternative hosts.

**Comment by everthemore** [comment]
@akhmerov are admin users the same as 'assistants'? (https://marketplace.zoom.us/docs/api-reference/zoom-api/users/userassistants), or do you think we should list all users and filter them by permissions (https://marketplace.zoom.us/docs/api-reference/zoom-api/users/userpermission)?

**Comment by akhmerov** [comment]
Assistants look good. This still means we need to assign scheduling privilege to LRC hosts, but that's still fine.

**Comment by akhmerov** [comment]
Actually, I'm looking at this part of the [documentation](https://support.zoom.us/hc/en-us/articles/201362803-Scheduling-Privilege) now:

> Users with Scheduling Privilege on your account will be able to manage and act as an alternative host for all meetings.

@greplova this means that you were trying to start the meeting without logging into your LRC account.

**Comment by akhmerov** [comment]
Shall we then add a checkbox to the LRC checklist?

**Comment by akhmerov** [comment]
Let's try to reproduce the issue manually.

**Comment by greplova** [comment]
Hi! So I tried now start and leave Org Meeting event (I was too terrified to accidentally meddle with LRC - but I think we agreed the two were exactly the same in the eyes of the vsf zoom account settings-wise)- and worked like a charm! So it must have been some strange fluke before. I noticed that my Zoom App got updated since the last attempt - that is the only difference I can identify between the two attempts. Bottom line now it works as it should. Thanks everyone for your help!
